README.md

```diff
@@ -91,6 +91,7 @@ These meetings occur weekly.
 - Platform - Wednesdays at 11:30am EST/EDT (15:30 UTC)
 - Implementation - Thursdays at 11:30am EST/EDT (15:30 UTC)
 - Buildpack Authors' Tooling - Friday at 10:00am EST/EDT (14:00 UTC)
+- Core - Wednesday at 2:00pm EST/EDT (18:00 UTC)
```

Comment:
nit: Core being at the bottom seems odd (given its importance). That said, to prevent ordering by "importance", I would like to propose that the meetings are listed alphabetically.

Reply:
Yeah, I just appended it to the bottom b/c I didn't want to usurp everything else. :) I'm :+1: +1: on alphabetical.